core/src/main/java/org/apache/james/mime4j/stream/RawFieldParser.java

```diff
@@ -323,6 +323,7 @@ public void skipAllWhiteSpace(final ByteSequence buf, final ParserCursor cursor)
      */
     public void copyContent(final ByteSequence buf, final ParserCursor cursor, final BitSet delimiters,
             final StringBuilder dst) {
+        ByteArrayBuffer dstRaw = new ByteArrayBuffer(80);
```

Comment:
why 80 bytes ?

Reply:
Likely because beyond 80 bytes headers needs to be folded so 80 seems safe at first sight but I agree we would need to test header lines longer than 80 bytes to be sure we do not accidentally ship a regression on marformed MIMEs

Reply:
IIRC it reallocates if necessary, so the initial size should be something that's large enough for many header fields (to avoid a large number of predictable reallocations) without wasting much space (allocating 80 bytes for a ten-byte Subject seemed OK, allocating 1k not so OK).

Reply:
(FWIW, I'm sick and I hate it. Ache to write code. Sigh.)